README.md

```diff
@@ -56,12 +58,56 @@ cd sage
 cargo run --release tests/config.json 
 ```
 
+## Installing the latest release
+
+1. Visit the [Releases](https://github.com/lazear/sage/releases/latest) website.
+2. Download the correct pre-compiled binary for your operating system.
+3. Run: `local/location/of/the/executable/sage config.json`
+
 # Usage 
 
-Sage takes a single command line argument: a path to a JSON-encoded parameter file (see below). 
+```shell
+$ sage --help
+Usage: sage [OPTIONS] <parameters> [mzml_paths]...
+
+🔮 Sage 🧙 - Proteomics searching so fast it feels like magic!
+
+Arguments:
+  <parameters>     The search parameters as a JSON file.
+  [mzml_paths]...  mzML files to analyze. Overrides mzML files listed in the parameter file.
+
+Options:
+  -f, --fasta <fasta>
+          The FASTA protein database. Overrides the FASTA file specified in the parameter file.
+  -o, --output_directory <output_directory>
+          Where the search and quant results will be written. Overrides the directory specified in the parameter file.
+  -h, --help
+          Print help information
+  -V, --version
+          Print version information
+```
+
+Sage is called from the command line using and requires a path to a JSON-encoded parameter file as an argument (see below). 
 
 Example usage: `sage config.json`
 
+Some options in the parameters file can be over-written using the command line
+interface. These are:
+
+1. The paths to the raw mzML data
+2. The path to the database (fasta file)
+3. The output directory
+
+like so ...
+
+```
+# specify fasta and output dir:
+sage -f proteins.fasta -o cool_project config.json
```

Comment:
Would you change this to `sage -f proteins.fasta -o output_directory config.json`?

Reply:
aggree